components/private/ShapeLibrary/src/MDCCornerTreatment+CornerTypeInitalizer.h

```diff
@@ -28,6 +28,17 @@
  */
 + (MDCRoundedCornerTreatment *)cornerWithRadius:(CGFloat)value;
 
+/**
+ Initialize and return an MDCCornerTreatment as an MDCRoundedCornerTreatment.
+
+ @param value The radius to set the rounded corner to.
+ @param valueType The value type in which the value is set as. It can be sent either as an
+ absolute value, or a percentage value (0.0 - 1.0) of the height of the surface.
+ @return an MDCRoundedCornerTreatment.
+ */
++ (MDCRoundedCornerTreatment *)cornerWithRadius:(CGFloat)value
+                                   andValueType:(MDCCornerTreatmentValueType)valueType;
```

Comment:
Suggestion: drop the `and` from the API names - I think this is a relatively uncommon pattern in Objective-C APIs.